docs/source/tutorials/traits_ui_scientific_app.rst

```diff
@@ -122,6 +122,8 @@ A point object could be implemented in python with:
     `code snippet #0 <../_static/code_block0.py>`_
 
     .. code-block:: python
+        
+        # code_block0.py
```

Comment:
instead of duplicating code, we should just be able to use the `code_block0.py` here. I think the way to do that is to use `literalinclude`

e.g. 

```
.. literalinclude:: path/to/scripy
   :language: python
```

Reply:
ref : https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude